primer_designer/templates/primer_designer/index.html

```diff
@@ -18,7 +39,7 @@
     <div class="card-body">
       <h2> Primer Designer</h2>
       <div>
-          Enter regions to design primers for (one region per line)
+          Enter regions to design primers for (one region per line - maximum 3)
```

Comment:
any reason for adding a limit?

Reply:
Set it to 3 because currently gunicorn is set for a timeout of 600 which is ~ the time to generate 4 primers. More primer new line > longer processing time > cause workers to timeout thus the limit.